packages/multi-trait-rubric/controller/src/defaults.js

```diff
@@ -1,16 +1,37 @@
 export default {
   model: {
-    visibleToStudent: true,
-    halfScoring: false,
-    pointLabels: true,
+    addScaleEnabled: true,
     description: false,
-    standards: false,
-    scales: [],
     excludeZero: false,
+    halfScoring: false,
     maxPointsEnabled: true,
-    addScaleEnabled: true,
+    pointLabels: true,
+    scales: [],
+    standards: false,
+    visibleToStudent: true,
   },
   configuration: {
```

Comment:
I think there's a mistake here, so it might be just the right time to fix this.

defaults from controller should only have `model`, not `configuration`, because the controller doesn't have access in real like to `configuration`.
Also, it seems like it's not being used properly in packages/multi-trait-rubric/controller/src/index.js, let's fix this:

`export function createDefaultModel(model = {}) { return new Promise((resolve) => resolve({ ...defaults, ...model })); }`
`export const normalize = (question) => ({ ...defaults, ...question });`